test/evergreen.yml

```diff
@@ -97,6 +97,21 @@ functions:
           set -o verbose
           echo '#undef HAVE_FTRUNCATE' >> wiredtiger_config.h
     - *make_wiredtiger
+  "compile wiredtiger with builtins":
```

Comment:
Just curious - do we use this compilation flag combination in other tests as well?

Reply:
Yes, other format-stress jobs  ppc, zseries can reuse this compile function.